pyquil/tests/test_examples.py

```diff
@@ -0,0 +1,62 @@
+
+import os
+import subprocess
+from subprocess import Popen, PIPE, STDOUT
+
+import pytest
+
+
+@pytest.fixture
+def examples_dir(scope="module"):
```

Comment:
a thing of beauty :)

Reply:
`scope="module"` should be an argument to `@pytest.fixture` instead of `examples_dir()`